src/alertsmanagement/azext_alertsmanagement/_params.py

```diff
@@ -0,0 +1,74 @@
+        c.argument('monitor_service', nargs='+', help='Filter alerts by monitor service')
+        c.argument('monitor_condition', nargs='+', help='Filter alerts by monitor condition')
+        c.argument('target_resource_type', nargs='+', help='Filter alerts by target resource type')
+        c.argument('alert_rule_id', nargs='+', help='Filter alerts by alert rule ID')
+        c.argument('alert_description', nargs='+', help='Filter alerts by alert rule description')
+        c.argument('alert_context', nargs='+', help='Filter alerts by alert context (payload)')
+        c.argument('recurrence_type', arg_type=get_enum_type(['Always', 'Once', 'Daily', 'Weekly', 'Monthly']), help='Specifies when the suppression should be applied')
+        c.argument('start_date', help='Start date for suppression')
+        c.argument('end_date', help='End date for suppression')
+        c.argument('start_time', help='Start time for suppression')
+        c.argument('end_time', help='End date for suppression')
+
+    with self.argument_context('alertsmanagement action-rule update') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('action_rule_name', action_rule_name)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('tags', tags_type)
+        c.argument('status', arg_type=get_enum_type(['Enabled', 'Disabled']), id_part=None, help='Indicates if the given action rule is enabled or disabled')
+
+    with self.argument_context('alertsmanagement action-rule delete') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('name', action_rule_name)
+
+    with self.argument_context('alertsmanagement action-rule show') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('name', action_rule_name)
+
+    with self.argument_context('alertsmanagement action-rule list') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('target_resource_group', id_part=None, help='Filter by target resource group name. Default value is select all.')
+        c.argument('target_resource_type', id_part=None, help='Filter by target resource type. Default value is select all.')
+        c.argument('target_resource', id_part=None, help='Filter by target resource( which is full ARM ID) Default value is select all.')
+        c.argument('severity', id_part=None, help='Filter by severity.  Default value is select all.')
+        c.argument('monitor_service', id_part=None, help='Filter by monitor service which generates the alert instance. Default value is select all.')
+        c.argument('impacted_scope', id_part=None, help='filter by impacted/target scope (provide comma separated list for multiple scopes). The value should be an well constructed ARM id of the scope.')
```

Comment:
should `impacted_scope` align with `scope` in creating parameters?

Reply:
It is deleted. Not in scope of this PR.